src/App.tsx

```diff
@@ -1,6 +1,7 @@
-import { Loader } from './components/Loader';
-
-import './App.scss';
+import { Routes, Route, Navigate, NavLink } from 'react-router-dom';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './components/PeoplePage';
```

Comment:
Ensure that the import path for `PeoplePage` is correct. It is currently imported from `./components/PeoplePage`, but it might be located elsewhere depending on your project structure.